development/architecture/migration-guide/horizontal-migration/migrate-list.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Cleaning up
+weight: 60
+---
+
+# Migrate list
+
+Once you create your controller,and you make your routing file. We can start to have fun with migration.
+
+## Routing
+We will start with the list. You have to add to the routing file an entry for the list.
+
+Keep our example with TitleController.
+
+Example:
+```yaml
+admin_title_index:
+  path: /
+  methods: [ GET, POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Configure\ShopParameters\TitleController::indexAction
+    _legacy_controller: AdminGender
+    _legacy_link: AdminGender
+```
+
+You must specify as
+
+- `_controller` the namespace of your controller
+- `_legacy_controller` the name of the legacy controller
+- `_legacy_link` the name of the legacy action (for the list it’s the same as the legacy controller)
```

Comment:
maybe worth briefly describing what are every one of these params used for and maybe provide some indication of what could go wrong (like some errors examples if some of them are missing)